manifests/manage_app_pool.pp

```diff
@@ -14,7 +14,8 @@
   $apppool_max_queue_length         = undef,
   $apppool_recycle_periodic_minutes = undef,
   $apppool_recycle_schedule         = undef,
-  $apppool_recycle_logging          = undef
+  $apppool_recycle_logging          = undef,
+  $apppool_idle_Timeout_action      = 'Terminate'
```

Comment:
We should default this to undef, and allow users to set it. Otherwise if someone upgrades the module it might change the behaviour if they'd already set it to suspend.